homeassistant/components/daikin/sensor.py

```diff
@@ -44,6 +45,8 @@ class DaikinRequiredKeysMixin:
     """Mixin for required keys."""
 
     value_func: Callable[[Appliance], float | None]
+    enabled_default: bool
```

Comment:
We don't need `enabled_default`. We already have the description attribute `entity_registry_enabled_default` and can change that for the descriptions.